sympy/solvers/bivariate.py

```diff
@@ -47,6 +47,10 @@ def _mostfunc(lhs, func, X=None):
     ``func`` can be a function (exp, log, etc...) or any other SymPy object,
     like Pow.
 
+    ``X`` has to be a varaible, by default which is `None`. If ``X`` is not `None`
+    only those terms will be selected that contains the variable specified
+    as ``X`` and the term returned would be one of these.
+
```

Comment:
> If `X` is not `None`, then the function returns the term composed with the most `func` having the  `specified variable`

I think `having the specified variable` should also be added because even if `X` is None it would return  the term composed with the most `func`. 
what do you suggest?